src/main/java/frc/utils/time/TimeUtils.java

```diff
@@ -14,6 +15,8 @@ public class TimeUtils {
 	private static double lastCycleTimeSeconds = 0;
 	private static double newCycleTimeSeconds = 0;
 
+	private static double currentTime;
```

Comment:
no as memvber

Reply:
?

Reply:
don't save it as member, save in the function to remove redundant calls for the same func

Reply:
fine